flowetl/Pipfile

```diff
@@ -0,0 +1,17 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+black = "==19.3b0"
+apache-airflow = "==1.10.3"
```

Comment:
Why's this pinned, out of interest?

Reply:
So that it is the same as the one in the container